index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portifolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <header>
+        <div class="header-toolbar-phone">
+            <a href="#" class="header-logo">Welcome</a>
+            <nav>
+                <button type="button" class="hamburger">
+                    <img src="images/humberger.svg" alt="menu link">
+                </button>
+            </nav>
+        </div>
+    </header>
+    <section class="headline">
+        <div class="header-title">
+            <h1>Hey there. I’m Mariane <span class="h-span">I’m a software developer</span></h1>
+        </div>
+        <div class="heder-info">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+        </div>
+        <div class="social-link">
+            <ul class="medias">
+                <li><a href=""><img src="images/twitter.png" alt=""></a></li>
+                <li><a href=""><img src="images/Linkedin icon.png" alt=""></a></li>
+                <li><a href=""><img src="images/medium.png" alt=""></a></li>
+                <li><a href=""><img src="images/github.png" alt=""></a></li>
+                <li><a href=""><img src="images/angellist.png" alt=""></a></li>
+            </ul>
+        </div>
+        <button class="head-btn" type="button"><img class="img-btn" src="images/Action.png" alt=""></button>
```

Comment:
**OPTIONAL** - This button isn't aligned in the center as it is in the Figma design template. Please fix this 👍. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
<img width="208" alt="project today" src="https://user-images.githubusercontent.com/73607512/198281313-caec47fc-8053-469f-87c5-e19a15ef8e48.PNG">

Design Template:
<img width="146" alt="button link figma" src="https://user-images.githubusercontent.com/73607512/198281489-8b264ec0-f58c-47aa-95ee-cc62309b97bc.PNG">